### DISCUSSION TITLE
Nifti images

### DISCUSSION THREAD

**Discussion by HripsimeS** [jcreinhold/intensity-normalization#60] [Q&A] [answered]
Hello @jcreinhold 

I saw your approach is working perfectly on MR images. Is it also working for Nifti or Dicom images ?

Thanks in advance!

**Comment by jcreinhold** [comment, marked as answer]
Yes, in fact MR images are usually stored in the DICOM or NIfTI file format.

**Reply by HripsimeS** [reply]
Thanks for your quick reply. I have CT images that are stored in NIfTI format, I assume it will work the same way as for MR images :)

**Reply by jcreinhold** [reply]
The code will execute on whatever images you provide it, but CT images don't need normalization; the intensities are already consistent across images, scanners, etc. (Hounsfield units).
